anonymize/anonymize.go

```diff
@@ -13,16 +16,34 @@ var adjacentSecrets = regexp.MustCompile(`xxx(\sxxx)+`)
 
 // Anonymize replace secret information with xxx.
 func Anonymize(src string, secrets ...string) (string, error) {
+	s, err := replaceNames(src)
+	if err != nil {
+		return src, errors.Wrapf(err, "fail to replace names in src %s", src)
+	}
 	tokens := tokenize(secrets...)
 	if len(tokens) == 0 {
-		return src, nil
+		return s, nil
 	}
 	secret, err := or(tokens)
 	if err != nil {
-		return src, err
+		return s, err
+	}
+	s = secret.ReplaceAllString(s, "xxx")
+	s = adjacentSecrets.ReplaceAllString(s, "xxx")
```

Comment:
This `xxx` is used 3 times in the code, please, move it to some constant.

Reply:
good point, ty!

Reply:
done